ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LT:
+  description: "Luminaire, lighting fixture."
+  implements:
+  - EQUIPMENT
+  uses:
+  - dimmer_percentage_command # in % indicating brightness
+
+LPS:
```

Comment:
Will this ever be modeled separately from a light? or a light controller? If not, it should be defined as a field instead.

In general, we should know what the atomic constructions are within the namespace. For example, from HVAC, we do not model subcomponents (e.g. coils are not modeled as separate pieces of equipment; they are sub-atomic in that case, and are represented with fields instead). Please comment on how/why you would want to model these things independently.

Reply:
@tasodorff please refer to [this document](https://docs.google.com/presentation/d/1u-eQqeKscl5VmFzQqHUHISr1nWVuUg-JrMm_jiOkwUw/edit#slide=id.g89fd74b19d_0_5) for a topology break down of lighting control systems. A lighting photocell sensor, or lighting motion sensor, or combined sensor can be 1) a discrete intelligent device like [this one](https://www.ivoryegg.co.uk/shop/products/basalte-180-02-auro-motion-detector-in-white) which has brightness, motion and temperature data points and is totally independent from a luminaire or 2) a little sensor device embedded in the luminaire like [this one](https://www.etaplighting.com/en/emd-multidetector) or 3) even a device attached to a lighting controller. In these three use cases, 1 and 3 need the specification of both an entity type for a device and then fields and subfields to describe the datapoints associated with it.